src/EditorFeatures/CSharpTest/DocumentationComments/CodeFixes/RemoveDocCommentNodeCodeFixProviderTests.cs

```diff
@@ -0,0 +1,606 @@
+}
+";
+            var parseOptions = Options.Regular.WithDocumentationMode(DocumentationMode.Diagnose);
+            await TestAsync(initial, expected, parseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveDuplicateParamTag)]
+        public async Task RemovesDuplicateParamTag_BothParamTagsOnSameLine_WhitespaceBetweenThem()
+        {
+            var initial =
+@"class Program
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name=""value""></param>     <param [|name=""value""|]></param>
+    public void Fizz(int value) {}
+}
+";
+
+            var expected =
+@"class Program
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name=""value""></param>
+    public void Fizz(int value) {}
+}
+";
+            var parseOptions = Options.Regular.WithDocumentationMode(DocumentationMode.Diagnose);
+            await TestAsync(initial, expected, parseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveDuplicateParamTag)]
+        public async Task RemovesDuplicateParamTag_BothParamTagsOnSameLine_NothingBetweenThem()
+        {
+            var initial =
+@"class Program
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name=""value""></param><param [|name=""value""|]></param>
+    public void Fizz(int value) {}
+}
+";
+
+            var expected =
+@"class Program
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <param name=""value""></param>
+    public void Fizz(int value) {}
+}
+";
+            var parseOptions = Options.Regular.WithDocumentationMode(DocumentationMode.Diagnose);
+            await TestAsync(initial, expected, parseOptions);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsRemoveDuplicateParamTag)]
+        public async Task RemovesDuplicateTypeparamTag()
+        {
+            var initial =
+@"class Program
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <typeparam name=""T""></typeparam>
+    /// <typeparam [|name=""T""|]></typeparam>
+    public void Fizz<T>() { }
+}
+";
+
+            var expected =
+@"class Program
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    /// <typeparam name=""T""></typeparam>
+    public void Fizz<T>() { }
+}
+";
+            var parseOptions = Options.Regular.WithDocumentationMode(DocumentationMode.Diagnose);
+            await TestAsync(initial, expected, parseOptions);
+        }
+
+        [Fact]
+        [Trait(Traits.Feature, Traits.Features.CodeActionsRemoveDuplicateParamTag)]
+        [Trait(Traits.Feature, Traits.Features.CodeActionsFixAllOccurrences)]
+        public async Task TestFixAllTypeparamInDocument_DoesNotFixDuplicateParamTags()
+        {
+            var initial = @"
+<Workspace>
+    <Project Language=""C#"" AssemblyName=""Assembly1"" CommonReferences=""true"" DocumentationMode=""Diagnose"">
+        <Document>
+class Program1
+{
+    /// &lt;summary&gt;
```

Comment:
Minor note: if you wrap the body in `<![CDATA[` and `]]>` tags you don't need `&lt;` and `&gt` which might make this easier to read.  E.g.,

``` C#
var initial = @"
<Workspace>
    <Project ...>
        <Document>
<![CDATA[
class Program 1
{
    /// <summary>
...
]]>
...
";
```